src/analysis/modals/ComputeModal/AzureComputeModal/AzureComputeModal.js

```diff
@@ -159,14 +165,28 @@ export const AzureComputeModalBase = ({
                   isSearchable: false,
                   isClearable: false,
                   value: computeConfig.machineType,
-                  onChange: ({ value }) => updateComputeConfig('machineType', value),
+                  onChange: ({ value }) => {
+                    onChangeComputeConfig(value);
+                  },
                   options: _.keys(azureMachineTypes),
                   getOptionLabel: ({ value }) => getMachineTypeLabel(value),
                   styles: { width: '400' },
                 }),
               ]),
             ]),
         ]),
+        hasGpu &&
+          div({ style: { display: 'flex', marginTop: '.5rem' } }, [
+            icon('warning-standard', { size: 16, style: { color: colors.warning(), marginRight: '0.5rem' } }),
+            h(
+              Link,
+              { href: 'https://support.terra.bio/hc/en-us/articles/16921184286491-How-to-use-GPUs-in-a-notebook-Azure-', ...Utils.newTabLinkProps },
+              [
+                'This VM is powered by an NVIDIA GPU. Learn more about enabling GPUs.',
```

Comment:
Hmm... if I was a user and I saw this I would not understand why it is a warning and probably ignore it. I'd like it if we could include something like `This VM is powered by an NVIDIA GPU. You may need to take external action to enable GPUs. Learn more.`

Reply:
I like that better too